src/System.Runtime/tests/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -2,50 +2,105 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Xunit;
 
 namespace System.ComponentModel.Tests
 {
     public static class DefaultValueAttributeTests
     {
         [Fact]
-        public static void TestCtor()
+        public static void TestCtor_Bool()
```

Comment:
Why separate this into different test methods?  Does it really make things more clear?  From my perspective, it makes it harder to see if anything is missing because the code ends up being 6x as long.